CI/circle_parallel.sh

```diff
@@ -57,8 +57,10 @@ elif [ "$NODE_INDEX" = "3" ]; then
   #cd Python-3.8.9
   #./configure --enable-optimizations
   #sudo make altinstall
+  pyenv --version
```

Comment:
The pyenv version that we have is `pyenv 1.1.5-2-g19cdbc8` and we need at least [1.2.21 or later](https://github.com/pyenv/pyenv/blob/master/CHANGELOG.md#1221) to install python 3.9.0 or higher. I am not seeing pyenv in apt-get
What would you like me to do here?
- Install all the pre-reqs and compile pyenv?
This is pretty involved per here: https://medium.datadriveninvestor.com/how-to-install-and-manage-multiple-python-versions-on-linux-916990dabe4b
- Or should we switch to a docker image? How would I do that as there are already 3 or 4 nodes
@wing328